.github/workflows/linters.yml

```diff
@@ -0,0 +1,46 @@
+name: Linters
+
+on:
+  pull_request:
+    branches: 
+     - "**"
+     
+env:
+  TARGET_BRANCH: ${{ github.base_ref }}
+
+jobs:
+  run-linters:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Run npm install
+      run: npm install --save-dev
+
+    - name: Fetch target branch
+      run: git fetch -n origin ${TARGET_BRANCH}
+            
+    - name: Run phpcs linter 
+      run: |
+        FILES_DIFF=$(git diff origin/${TARGET_BRANCH} --no-commit-id --name-only -r -- '*.php')
+        if [ -n "$FILES_DIFF" ]; then
+          ./bin/phpcs.phar $FILES_DIFF --standard=phpcs-ruleset.xml
+        fi
+
+    - name: Run stylelint linter
+      if: ${{ always() }}
```

Comment:
Please add a comment here and on the next step to explain why we're doing this

Reply:
@davialexandre  Comment is added.